src/main/resources/application.yaml

```diff
@@ -187,3 +187,16 @@ ai:
   api_key:      "sk-api-key"
   request_body: '{ "input": "$message"}'
   message_replacement_pattern: "$message"
+
+#
+# Google Auth Properties
+#
+google:
+  auth:
+    server:
+      url: "https://accounts.google.com/o/oauth2/v2/auth"
+  scope: "https://www.googleapis.com/auth/userinfo.profile"
+  #client-id: "sk-client-id"
+  #client-secret: "sk-client-secret"
+  authenticateUri: "https://iced-latte.uk/auth/login"
```

Comment:
Did you mean `/auth/authenticate` endpoint?
I don't see any mention of `/auth/login` in specs 🤔

Reply:
need frontend endpoint, emulate user login